pandas/io/excel.py

```diff
@@ -40,17 +40,16 @@
 _writers = {}
 
 _read_excel_doc = """
-Read an Excel table into a pandas DataFrame
+Read an Excel table into a pandas DataFrame.
 
 Parameters
 ----------
-io : string, path object (pathlib.Path or py._path.local.LocalPath),
-    file-like object, pandas ExcelFile, or xlrd workbook.
+io : str, path object (pathlib.Path or py._path.local.LocalPath), \
+     file-like object, pandas ExcelFile, or xlrd workbook
```

Comment:
We try to use the Python file types comma separated (I'd like to be able to parse these types). This would make more sense to me:

`io : str, file descriptor, pathlib.Path, ExcelFile or xlrd.Book`

In the description you can add further information if needed.

Reply:
Since parameter can be a "pathlib.Path or py._path.local.LocalPath" would it make sense to make the descriptor more general ("path")? I see that the Py library is in maintenance mode, just seeking to understand going forward!